DisCatSharp/Entities/Channel/DiscordChannel.cs

```diff
@@ -127,7 +127,7 @@ public int GetMinPosition()
 	/// </summary>
 	[JsonIgnore]
 	public bool IsPrivate
-		=> this.Type == ChannelType.Private || this.Type == ChannelType.Group;
+		=> this.Type is ChannelType.Private or ChannelType.Group or ChannelType.PrivateThread;
```

Comment:
IsPrivat is to determine whether it's a dm in this context

Reply:
Can we do an override of that function maybe

Reply:
This would in effect be equivalent to the original change of adding `PrivateThread` to the valid `IsPrivate` types which is not what you want

Reply:
Or wait.. no we don't need it. 
I remember now why I kept it smh. In the past it was possible to create threads in dms haha